.github/ISSUE_TEMPLATE/new-module-proposal.md

```diff
@@ -0,0 +1,32 @@
+---
+name: New module proposal
+about: Propose a new module for the Performance Lab plugin
+title: 'Module Proposal: '
+labels: Needs Discussion, no milestone, [Type] Module
+assignees: ''
```

Comment:
See above, let's remove this since it's just the default.

Reply:
Same as above, will need to play wit this more.